clients/common/pom.xml

```diff
@@ -32,6 +32,19 @@
 
   <dependencies>
     <dependency>
+      <groupId>org.glassfish</groupId>
```

Comment:
I'm not 100% sure if it's the right place for that (in clients/common, scope provided)... is it ok?